runner/config/development-config.yaml

```diff
@@ -15,6 +15,8 @@ rest_key:
 log_dir: /tmp/shift/
 pt_osc_path: pt-online-schema-change
 pending_drops_db: _pending_drops
+log_sync_interval: 10
```

Comment:
update https://github.com/square/shift/tree/master/runner#configuration to describe these config options